Sources/SentryCrash/Recording/SentryCrashReport.c

```diff
@@ -913,14 +913,16 @@ writeNotableStackContents(const SentryCrashReportWriter *const writer,
         highAddress = tmp;
     }
     uintptr_t contentsAsPointer;
-    char nameBuffer[40];
+#define SENTRY_NAME_BUFFER_SIZE 40
+    char nameBuffer[SENTRY_NAME_BUFFER_SIZE];
     for (uintptr_t address = lowAddress; address < highAddress; address += sizeof(address)) {
         if (sentrycrashmem_copySafely(
                 (void *)address, &contentsAsPointer, sizeof(contentsAsPointer))) {
-            sprintf(nameBuffer, "stack@%p", (void *)address);
+            snprintf(nameBuffer, SENTRY_NAME_BUFFER_SIZE, "stack@%p", (void *)address);
```

Comment:
`m`: I would prefer adding a test to cover these lines of code to ensure we are not causing any crashes.

Reply:
oof, the other one was pretty simple to write a test case for, but this one is tougher. so far we have no coverage of this entire function, and it's gonna be fun to mock up a fixture: it will basically be an entire description of a stack trace formatted in the structs used by the crash reporter. will work more on this.

Reply:
I'm going to back out this change and only leave in the one that I already covered in test. This one is too big of a lift right now.